package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.40.0",
+  "version": "2.40.1",
```

Comment:
Included in the PR because this might be released early due to urgent customer requirements.